### ISSUE TITLE
Release crates.io patch to avoid security vulnerability

### ISSUE THREAD

**Issue by trevor-crypto** [graphql-rust/graphql-client#425]
Not exactly an issue, but I noticed that there has not been a release on crates.io in a while. Is it possible to release a new version there? It would remove the following security vulnerability since the `failure` crate has since been removed from the dependency graph. Thanks.

https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25575
```
 graphql_client v0.10.0
   ├── graphql_query_derive v0.10.0 (proc-macro)
   │   ├── graphql_client_codegen v0.10.0
   │   │   ├── graphql-introspection-query v0.2.0
   │   │   │   └── serde v1.0.136 (*)
   │   │   ├── graphql-parser v0.2.3
   │   │   │   ├── combine v3.8.1
   │   │   │   │   ├── ascii v0.9.3
   │   │   │   │   ├── byteorder v1.4.3
   │   │   │   │   ├── either v1.6.1
   │   │   │   │   ├── memchr v2.4.1
   │   │   │   │   └── unreachable v1.0.0
   │   │   │   │       └── void v1.0.2
   │   │   │   └── failure v0.1.8
```

**Comment by tomhoule** [comment]
I don't have time to work for free on this project. I released 0.11.0 but I can't guarantee any further work. If someone is interested in sharing maintainership work, please step up.

**Comment by trevor-crypto** [comment]
@tomhoule Thank you.
